CHANGES.md

```diff
@@ -1,4 +1,8 @@
-### 0.1.7 20202-01-09
+### Unreleased
+
+- Remove socat dependency in debug-socket script.
```

Comment:
If you want this released, you can bump the version in this PR and I will release it.

Reply:
Thanks. I just bumped the version in the most recent commit.